cypress/integration/common/index.js

```diff
@@ -193,3 +193,7 @@ And('the following delighters are displayed', (dataTable) => {
 Then('help icon is displayed with href {string}', (helpHref) => {
 	cy.get('a.text-icon-help').should('have.attr', 'href', helpHref);
 });
+
+Then('search is not executed and path is {string}', (path) => {
```

Comment:
can this please be updated to "**the** search is not executed..." so as to be similar to "the search is executed and results page is displayed"

It may stay in index.js as I have been explained why. (Although merge conflicts will probably happen)